src/test/java/GuavaProgramming/StringProcessingUtilityTest.java

```diff
@@ -0,0 +1,49 @@
+package GuavaProgramming;
+
+import com.google.common.collect.Lists;
+import com.google.common.collect.Maps;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Created by saurav on 9/7/16.
+ */
+public class StringProcessingUtilityTest {
+    StringProcessingUtility sp;
+    @Before
+    public void setUp() throws Exception {
+        sp = new StringProcessingUtility();
```

Comment:
Actually u can skip this and initialize the object beginning. 
final StringProcessingUtility sp = new StringProcessingUtility();